requirements.txt

```diff
@@ -47,8 +47,8 @@ suds==0.4 \
     --hash=sha256:d5524ae523bb9e09c57bcdbd1efae2c287d20603688ea31f6020ed180a489af0
 https://github.com/jbalogh/test-utils/archive/e42e031.zip#egg=test-utils \
     --hash=sha256:8a1dbd1c7ade85206ae770c9ce5f43081a098c95f994af113f6434e900559a67
-translate-toolkit==1.13.0 \
-    --hash=sha256:dcbbf49058e4196a06e988d9dc1e762321ab0d057c4be035b84e3c11353fc2f8
+https://github.com/mathjazz/translate/archive/1.13.1.zip#egg=translate==1.13.1 \
+    --hash=sha256:0130db76e886f26ba0d6dc69c1ef2659315ea972af4aa95cda69c1e53f322707
```

Comment:
We need to use a custom fork before the fix proposed upstream gets released:
https://github.com/translate/translate/issues/3424